src/StructuredLogger.Tests/BinaryLoggerTests.cs

```diff
@@ -11,7 +11,11 @@
 using Xunit;
 using Xunit.Abstractions;
 using static StructuredLogger.Tests.TestUtilities;
+using ArchiveFileEventArgs = Microsoft.Build.Logging.StructuredLogger.ArchiveFileEventArgs;

```

Comment:
what are these for?

Reply:
Without it compiler gets lost because ArchiveFileEventArgs is present in two different namespaces